serde/src/java/org/apache/hadoop/hive/serde2/objectinspector/StandardConstantMapObjectInspector.java

```diff
@@ -40,8 +40,8 @@ protected StandardConstantMapObjectInspector() {
    */
    protected StandardConstantMapObjectInspector(ObjectInspector mapKeyObjectInspector,
        ObjectInspector mapValueObjectInspector, Map<?, ?> value) {
-    super(mapKeyObjectInspector, mapValueObjectInspector);
-    this.value = value;
+     super(mapKeyObjectInspector, mapValueObjectInspector);
```

Comment:
![gif](https://media.giphy.com/media/3FjEPbKqEPhPpmC8uY/giphy.gif)

I think at some point we will add rules to enforce coding style...but until then:
we can have these 2 new spaces in the Hive codebase - we can't have enough of them :)

Reply:
Looking forward into master branch.